### ISSUE TITLE
Scroll bar doesn't match the area to scroll

### ISSUE THREAD

**Issue by Haroenv** [bullgit/chitchat#36]
It goes over the complete height of the page, not just the messages part.

**Comment by LukyVj** [comment]
Yeah I noticed that too, I think I added some interesting changes, I have to push them, if you wanna fix it, feel free :)

**Comment by Haroenv** [comment]
I don't have time right now, maybe during next week
